src/Stripe.net/Infrastructure/Middleware/ParserPlugin/ListPlugin.cs

```diff
@@ -27,8 +36,9 @@ public bool Parse(ref string requestString, JsonPropertyAttribute attribute, Pro
                     // it must have a json attribute matching stripe's key, and only one
                     var attr = prop.GetCustomAttributes<JsonPropertyAttribute>().SingleOrDefault();
                     if (attr == null) continue;
+
                     // a JsonPropertyAttribute is required to supply a property name to parser plugins.
-                    var indexedItemAttribute = new JsonPropertyAttribute($"items[{itemIndex}][{attr.PropertyName}]");
+                    var indexedItemAttribute = new JsonPropertyAttribute($"{attribute.PropertyName}[{itemIndex}][{attr.PropertyName}]");
```

Comment:
minor but `attribute` versus `attr` in this line is a bit confusing/hard to read

Reply:
👍  I copied/pasted from the previous plugins but we should take this opportunity to clarify the code. I'll update with better variable names.